WNPRC_Purchasing/test/src/org/labkey/test/Pages/CreateRequestPage.java

```diff
@@ -33,7 +33,19 @@ public String getAccountsToCharge()
 
     public CreateRequestPage setAccountsToCharge(String option)
     {
-        elementCache().accountToCharge.selectByVisibleText(option);
+        //Retrying for ajax to load the options.
+        int attempts = 0;
+        while (attempts < 3)
+        {
+            try
+            {
+                elementCache().accountToCharge.selectByVisibleText(option);
+            }
+            catch (NoSuchElementException e)
+            {
+                attempts++;
+            }
+        }
```

Comment:
This will loop forever if the selection succeeds and fail silently if the selection doesn't succeed within the attempt limit. Explicitly waiting for the option is clearer and less prone to bugs:
```suggestion
        Locator.tag("option").withText(option).waitForElement(elementCache().accountToCharge, 5_000);
        elementCache().accountToCharge.selectByVisibleText(option);
```